dev/npm.ts

```diff
@@ -1,50 +1,37 @@
+const searchGenerator: Fig.Generator = {
+  script: function (context) {
+    if (context[context.length - 1] === "") return "";
+    const searchTerm = context[context.length - 1];
+    return `curl -s -H "Accept: application/json" "https://api.npms.io/v2/search?q=${searchTerm}&size=250"`;
+  },
+  postProcess: function (out) {
+    return JSON.parse(out).results.map(
+      (item) => item.package.name
+    ) as Fig.Suggestion[];
```

Comment:
The json also includes the description. I think this would be extremely nice if we could also include it

```suggestion
    return JSON.parse(out).results.map(
      (item) =>
        ({
          name: item.package.name,
          description: item.package.description,
        } as Fig.Suggestion)
    ) as Fig.Suggestion[];
```

Reply:
The `Fig.Suggestion[]` should be inferred by typescript since we cast above in the map to a `Fig.Suggestion`, isn't it?

Reply:
Unfortunately it isn't. Atleast I didn't get autocompletion in vscode